src/docfx/template/MustacheTemplate.cs

```diff
@@ -64,6 +70,25 @@ private static RendererSettingsBuilder UseJson(RendererSettingsBuilder settings,
 
                 return childToken is JValue scalar ? scalar.Value ?? JValue.CreateNull() : childToken;
             }
+
+            object? GetJArrayValue(object value, string key, bool ignoreCase)
+            {
+                _ = ignoreCase;
+                var jArr = (JArray)value;
+                switch (key.ToLowerInvariant())
+                {
+                    case "length":
+                        return jArr.Count.ToString();
+                    default:
+                        return int.TryParse(key, out var index) && index > -1 && jArr.Count > index
+                            ? jArr[index] is JValue scalar
+                                ? scalar.Value ?? JValue.CreateNull()
+                                : jArr[index]
+                            : JValue.CreateNull();
```

Comment:
Will `null` work? The function should return `JObject`, `JArray` or `primitive type`, `JValue.CreateNull()` is not a primitive type.
```suggestion
                            : null;
```

Reply:
Let's take a try

Reply:
I didn't touch it due to this PR
https://github.com/dotnet/docfx/pull/4932/files#diff-e57f7b911be9c3fec40a3f7ccc3f17b4R62